exchanges/stream/websocket.go

```diff
@@ -683,6 +679,34 @@ func (w *Websocket) FlushChannels() error {
 	return nil
 }
 
+// updateChannelSubscriptions subscribes or unsubscribes from channels and checks that the correct number of channels
+// have been subscribed to or unsubscribed from.
+func (w *Websocket) updateChannelSubscriptions(c Connection, store *subscription.Store, incoming subscription.List) error {
+	if store == nil {
+		return fmt.Errorf("%w for %T", common.ErrNilPointer, store)
+	}
```

Comment:
Nit: Since this is a private method, I'd hope we wouldn't need this check, and associated tests.